src/convert.rs

```diff
@@ -7,13 +20,13 @@ macro_rules! convert {
         impl Convert<$b> for $a {
             #[inline(always)]
             fn convert(self) -> $b {
-                zerocopy::transmute!(self)
+                unsafe { safer_transmute!(self) }
```

Comment:
This still suffers from the fact that the `unsafe` is hidden in a macro, and also there is no SAFETY comment. I think those were the two problems that the PR that added zerocopy was trying to address.

If it is important to use `transmute` for performance or other reasons, then it would be better to just manually copy/paste the necessary implementations so that all the `unsafe` are moved out of the macro.

Reply:
Agreed. This would need to be addressed.  I just saw that you pushed PRs up that I think in general are a good thing to land if possible.  So maybe that should take priority.

I general though I think all of this is stalled for the moment until a maintainer shows up.  I'm not sure who has permissions to merge or approve things here.